src/WebJobs.Script/Environment/EnvironmentSettingNames.cs

```diff
@@ -101,5 +101,12 @@ public static class EnvironmentSettingNames
        // EasyAuth settings
         public const string EasyAuthClientId = "WEBSITE_AUTH_CLIENT_ID";
         public const string EasyAuthSigningKey = "WEBSITE_AUTH_SIGNING_KEY";
+
+        // Antares version settings
+        public const string AntaresVersionWindows = "WEBSITE_PLATFORM_VERSION";
+        public const string AntaresVersionLinux = "PLATFORM_VERSION";
+
+        // Machine identifier
+        public const string MachineIdentifier = "WEBSITE_INSTANCE_ID";
```

Comment:
Reuse EnvironmentSettingNames.AzureWebsiteInstanceId?

Reply:
Yeah, I realized that later and changed it already.